contract_queue_job/models/contract_line.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+from odoo.addons.queue_job.job import job
+
+QUEUE_CHANNEL = "root.CONTRACT_INVOICE"
```

Comment:
IMO, it's better to do it on a specific channel 

```suggestion
QUEUE_CHANNEL = "root.CONTRACT_LINE_RENEW"
```